src/front/js/component/Navbar.jsx

```diff
@@ -1,20 +1,57 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import starwarsImageUrl from "../../img/logo-starwars.png";
 
 export const Navbar = () => {
-	
-	return (
-		<nav className="navbar navbar-light bg-light">
-			<div className="container">
-				<Link to="/">
-					<span className="navbar-brand mb-0 h1">React Boilerplate</span>
-				</Link>
-				<div className="ml-auto">
-					<Link to="/demo">
-						<button className="btn btn-primary">Check the Context in action</button>
-					</Link>
-				</div>
-			</div>
-		</nav>
-	);
+
+  return (
+    <nav className="navbar navbar-expand-lg navbar-dark bg-dark" aria-label="Offcanvas navbar large">
+      <div className="container-fluid">
+        <Link to="/">
+          <a className="navbar-brand" href="/"><img className="mx-4" height="70" src={starwarsImageUrl} /></a>
```

Comment:
@AnaPaez89 
Elimina la etiquetas `<a>` que solo queden las etiquetas `<Link to="/" className="navbar-brand">`